api/src/certification/session-management/domain/read-models/JuryCertificationSummary.js

```diff
@@ -23,7 +27,10 @@ class JuryCertificationSummary {
     this.status = _getStatus({ status, isEndedBySupervisor });
     this.pixScore = pixScore;
     this.isFlaggedAborted = Boolean(abortReason) && !completedAt;
-    this.complementaryCertificationTakenLabel = complementaryCertificationTakenLabel;
+    this.certificationTaken = _getCertification({
```

Comment:
```suggestion
    this.currentCertification = _getCertification({
```

Je sais pas pourquoi j'ai du mal avec ce `taken` 😅

Reply:
C'est la traduction de "certification passée" par le candidat. 
`current` ne me semble pas plus approprié :/

Reply:
`certificationObtained` ?